modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/MeResourceManagerTest.java

```diff
@@ -0,0 +1,1547 @@
+            "    ],\n" +
+            "\"meta\": {\n" +
+            "    \"created\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"location\": \"ENDPOINT/008bba85-451d-414b-87de-c03b5a1f4217\",\n" +
+            "    \"lastModified\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"resourceType\": \"User\"\n" +
+            "},\n" +
+            "  \"Operations\": [\n" +
+            "    {\n" +
+            "      \"op\": \"replace\",\n" + //REPLACE Operation
+            "      \"value\": {\n" +
+            "        \"nickName\": \"shaggy\"\n" +
+            "      }\n" +
+            "    }\n" +
+            "  ],   \n" +
+            " \"name\": {\n" +
+            "    \"givenName\": \"Kim\",\n" +
+            "    \"familyName\": \"Berry\",\n" +
+            "    \"formatted\": \"Kim Berry\"\n" +
+            "  },\n" +
+            "  \"userName\": \"kimjohn  \",\n" +
+            "  \"password\": \"kim123\",\n" +
+            "  \"id\": \"123\",\n" +
+            "  \"emails\": [\n" +
+            "      {\n" +
+            "        \"type\": \"home\",\n" +
+            "        \"value\": \"john@gmail.com\",\n" +
+            "         \"primary\": true\n" +
+            "      }\n" +
+            "  ]\n" +
+            "}";
+
+    private static final String NEW_USER_SCIM_OBJECT_STRING_FOR_PATCH_WITH_REPLACE_AND_UPDATED = "{\n" +
+            "  \"schemas\": \n" +
+            "    [\"urn:ietf:params:scim:schemas:core:2.0:User\",\n" +
+            "    \"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User\",\n" +
+            "    \"urn:ietf:params:scim:api:messages:2.0:PatchOp\"\n" +
+            "    ],\n" +
+            "\"meta\": {\n" +
+            "    \"created\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"location\": \"ENDPOINT/008bba85-451d-414b-87de-c03b5a1f4217\",\n" +
+            "    \"lastModified\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"resourceType\": \"User\"\n" +
+            "},\n" +
+            "  \"Operations\": [\n" +
+            "    {\n" +
+            "      \"op\": \"replace\",\n" + //REPLACE Operation
+            "      \"value\": {\n" +
+            "        \"nickName\": \"shaggy\"\n" +
+            "      }\n" +
+            "    }\n" +
+            "  ],   \n" +
+            " \"name\": {\n" +
+            "    \"givenName\": \"Kim\",\n" +
+            "    \"familyName\": \"Berry\",\n" +
+            "    \"formatted\": \"Kim Berry\"\n" +
+            "  },\n" +
+            "  \"userName\": \"kimjohn  \",\n" +
+            "  \"password\": \"kim123\",\n" +
+            "  \"id\": \"123\",\n" +
+            "  \"emails\": [\n" +
+            "      {\n" +
+            "        \"type\": \"home\",\n" +
+            "        \"value\": \"johnnew@gmail.com\",\n" +
+            "         \"primary\": true\n" +
+            "      }\n" +
+            "  ]\n" +
+            "}";
+
+    private static final String ENDPOINT_ME = "https://localhost:9443/scim2/Me";
```

Comment:
```suggestion
    private static final String SCIM2_ME_ENDPOINT = "https://localhost:9443/scim2/Me";
```

Reply:
Fixed in commit a5699e694bc31369fb4a347d4a0a12177e73775d